src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/WebJira.kt

```diff
@@ -94,6 +94,20 @@ data class WebJira(
         return driver.findElement(By.id("footer-build-information")).text
     }
 
+    fun goToFocusedComment(
+        commentUrl: String
+    ): CommentTabPanel {
+        driver.navigate().to(commentUrl)
+        return CommentTabPanel(driver)
+    }
+
+    fun goToHistoryTab(
+        issueKey: String
+    ): HistoryTabPanel {
+        navigateTo("/browse/$issueKey?page=com.atlassian.jira.plugin.system.issuetabpanels:changehistory-tabpanel")
```

Comment:
Topic for discussion, possibly not as part of this PR:

Should we glue such complex URLs in the action code? I understand that users may do so for simple ones like `<Jira>/browse/<issue_key>` or `<Jira>/issues/?jql=<JQL>` (at least I sometimes do), but are people adding this `page=com.atlassian.jira.plugin.system.issuetabpanels:changehistory-tabpanel` URL suffix instead of clicking the tab?

The same goes for other existing `navigateTo()` in this class so I'm not for changing the approach in this PR.

On the other hand static URLs like `secure/Dashboard.jspa` can be bookmarked. That saying issue view with history tab can also be bookmarked, however then we should probably remember it as a whole in memory (or if there is some contract around construction of those URLs maybe it would be good to extract it to separate responsibility class).

Reply:
I agree. I was thinking about having View History Tab Action implemented as an extension to View Issue Action. Then, while being on issue view we could just click on the tab and measure the loading time but with this approach it would be hard to separate this actions from the existing ones. Maybe not in this PR, but later I'll come up with some improvement suggestions.

Reply:
Clicking on a tab can be inlined in the `ViewIssueAction` with a given probability.
Years ago, I used a side-repo for Cloud vs Server benchmark, which has such probabilistic sub-actions, like [WorkOnIssue](https://github.com/atlassian/jces-1209/blob/master/custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnIssue.kt). Props to @atlassian/jpt-cloud team for extending the coverage so much :heart: 
It has an extra abstraction to hide the differences between Server and Cloud, but we could actually harvest that too (and potentially expand for DC or Jira 8/9).

Reply:
This was addressed. Closing the converastion.